File: src/main/java/frc/robot/commands/cargo/IntakeCargo.java
package frc.robot.commands.cargo;

import edu.wpi.first.wpilibj.command.CommandGroup;
import frc.robot.subsystems.ClawAndIntake;
import frc.robot.subsystems.MouthArticulator;
import org.apache.logging.log4j.LogManager;
import org.apache.logging.log4j.Logger;

public class IntakeCargo extends CommandGroup {

    private final Logger LOG = LogManager.getLogger(IntakeCargo.class);

    public IntakeCargo() {
        addParallel(new ArticulateMouth(MouthArticulator.MouthPosition.OUT));
        addSequential(new RunClaw(ClawAndIntake.ClawState.INTAKING));
        setName("IntakeCargo CommandGroup");
        LOG.info(getName() + " Constructed");
    }
}

File: src/main/kotlin/ktfrc/robot/commands/cargo/IntakeCargo.kt
package ktfrc.robot.commands.cargo

import edu.wpi.first.wpilibj.command.CommandGroup
import ktfrc.robot.subsystems.ClawState
import ktfrc.robot.subsystems.MouthArticulatorPosition
import org.apache.logging.log4j.LogManager
import org.apache.logging.log4j.Logger

class IntakeCargo: CommandGroup() {

    init {
        addParallel(ArticulateMouth(MouthArticulatorPosition.OUT))
        addSequential(RunClaw(ClawState.INTAKING))
        name = "IntakeCargo CommandGroup"
        LOG?.info("$name Constructed")
    }

    companion object {
        val LOG: Logger? = LogManager.getLogger(IntakeCargo::class.java)
    }
}